### DISCUSSION TITLE
AccurateRip log

### DISCUSSION THREAD

**Discussion by Mnemon76** [enzo1982/freac#271] [Help & Support] [answered]
I have previously used the cuetools and cueripper on Windows,  I am now using the latest Puppy Linux and fre:ac to rip a cd collection.  I am interested in the Acurrate rip verification step and how this is implemented.  In the cueripper software an Accurate Rip log file is created in the folder the flac files are placed.

I also am not sure which settings are affecting the track filenames:

Filename "Tiny Music... " becomes "Tiny Music#"

Thanks

**Comment by enzo1982** [comment, marked as answer]
AccurateRip results are included in the regular log as shown in the _Logs_ tab of the main window.

You can also configure fre:ac to save these logs with the output files by enabling this in the config dialog under _Extensions->Logging_.

The three dots being replaced may be due to an option in fre:ac. In the config dialog under _Encoders->Output filenames_, check if _"Allow Unicode characters"_ is enabled. It's enabled by default, but maybe you changed that earlier.

When that option is disabled, any Unicode characters in the output filename will be replaced by a hashtag (#). This would happen if the ellipsis in your song title is actually a Unicode ellipsis character ([U+2026](https://codepoints.net/U+2026)) instead of three individual dots.

Otherwise, filenames are generated according to the filename pattern configured in the same section. You can find more information about the available placeholders at: https://freac.org/manual/en/howto.html#anchor_pat

**Reply by Mnemon76** [reply]
Thanks for the reply. I got the filename issue sorted somehow. It may have been the Unicode checkbox or my filesharing between Windows and Linux not working correctly.

I have a question about the ripping operation and a comment. According to this Wiki fre:ac is still not shown as implementing AccurateRip.

[https://wiki.hydrogenaudio.org/index.php?title=Comparison_of_CD_rippers](url)

On that same comparison chart I noticed that "Gap Detection" and "C2 Pointers" were a "NO" vs the "YES" on cueripper. I am wondering if  that is part of the reason fre:ac is so much faster ripping than cueripper on my Windows machine. I also have a much slower USB drive on the Windows machine. 

I like fre:ac on the Linux machine, I am just trying to understand the differences as I believe cuetools is also based on cdparanoia, however it has a "secure" setting as well as a "test & copy" option. It takes multiple passes at some of by old CDs but manages to rip them well according to the AccurateRip database. It seems slow the drive down significantly at times.

Tomorrow I will attach a couple log files I am trying to understand why the cuetools program fails to verify the rip against the AccurateRip DB the same way fre:ac does.

Thank you,

**Reply by enzo1982** [reply]
fre:ac is correctly listed as supporting AccurateRip, but not supporting *"checking across pressings/offsets"* on the HydrogenAudio Wiki page.

The checking across offsets feature allows successful verification even if your disc is a different pressing with a slightly different sample offset than the one in the AccurateRip database. I plan to add support for this in a future version of fre:ac.

fre:ac not supporting gap detection and C2 pointers is correct. And yes, that might play a role in fre:ac being faster than cueripper. I would like to add support for these features too at some point, but it's not foreseeable yet when I will get to it. A *test & copy* mode will likely be available earlier.

Generally, if fre:ac or any other application successfully verifies a rip against the AccurateRip database, the rip will be fine. If a track fails to verify, you can try again - some types of errors may go away on a second try.

If a disc is not in the AccurateRip database, you may want to use a tool that offers C2 checking (though your drive needs to support that too) and/or test & copy.

If fre:ac successfully verifies a rip, but cuetools fails to do so, you likely have a wrong drive offset set in cuetools.
